velox/common/memory/Memory.cpp

```diff
@@ -271,8 +273,7 @@ std::shared_ptr<MemoryPool> MemoryManager::addRootPool(
       options);
   pools_.emplace(poolName, pool);
   VELOX_CHECK_EQ(pool->capacity(), 0);
-  arbitrator_->growCapacity(
-      pool.get(), std::min<uint64_t>(poolInitCapacity_, maxCapacity));
```

Comment:
Before this patch, we could allocate poolInitCapacity_ and then grow it when needed, but now ,we will allocate maxCapacity for it, is it your intension?